cpu/esp32/startup.c

```diff
@@ -121,16 +137,16 @@ NORETURN void IRAM call_start_cpu0 (void)
 {
     register uint32_t *sp __asm__ ("a1"); (void)sp;
 
-    cpu_configure_region_protection();
+    esp_cpu_configure_region_protection();
 
     /* move exception vectors to IRAM */
-    asm volatile ("wsr %0, vecbase\n" ::"r"(&_init_start));
+    asm volatile ("wsr %0, vecbase\n" ::"r"(&_iram_start));
 
     RESET_REASON reset_reason = rtc_get_reset_reason(PRO_CPU_NUM);
 
     /* reset from panic handler by RWDT or TG0WDT */
     if (reset_reason == RTCWDT_SYS_RESET || reset_reason == TG0WDT_SYS_RESET) {
-        esp_panic_wdt_stop();
+        /* TODO esp_panic_wdt_stop was called here in former versions */
```

Comment:
Is this no longer needed?

Reply:
It's just a leftover. The bootloader starts a watchdog that was stopped by the user application in `startup` function in former IDF version. This is now an option that is configured by `CONFIG_BOOTLOADER_WDT_DISABLE_IN_USER_CODE`. If this option isn't set (the default), the watchdog isn't disabled at startup. Since we disable the RTC flashboot watchdog in `system_wdt_init`, it is not necessary to stop it here.

Reply:
So why the TODO?

Reply:
Since I wasn't sure when I started with ESP-IDF v4.4.